src/neural/factory.cc

```diff
@@ -119,7 +119,12 @@ std::unique_ptr<Network> NetworkFactory::LoadNetwork(
   OptionsDict network_options(&options);
   network_options.AddSubdictFromString(backend_options);
 
-  return NetworkFactory::Get()->Create(backend, weights, network_options);
+  auto ptr = NetworkFactory::Get()->Create(backend, weights, network_options);
+
+  auto x = network_options.FindNotUsed();
+  if (x.size()) throw Exception("Unknown backend option \"" + x + "\"");
```

Comment:
Why not `(!x.empty())`? Seems more aligned with intent.

Reply:
First thing that came to mind and never revised - will change it.